src/components/navigation/right-nav.sass

```diff
@@ -63,43 +63,107 @@ $list-item-scale: 0.11
       position: relative
       width: 100%
       height: 100%
+      border-radius: $border-radius 0 0 $border-radius
+      padding: 13px
+      background-color: $color5-5
+      overflow: auto
+      overflow-x: hidden
 
       .disabled
         display: none
 
       .loading
         padding: 10px
 
+      ::-webkit-scrollbar
+        width: 7px
+      ::-webkit-scrollbar-track
+        background: #f1f1f1
+      ::-webkit-scrollbar-thumb
+        background: $color3
+        border-radius: $border-radius
+      ::-webkit-scrollbar-thumb:hover
+        background: #a3a3a3
```

Comment:
A couple of notes here.  1. this pseudo element will only currently work on Chrome and Safari (not Edge or Firefox).  For our use, it will probably do, but we should consider a hand-rolled scrollbar if we want to support all browsers. 2. I'm keeping this to the right nav, but I'm willing to bet that if we looked through the spec this might apply to all scrollbars throughout the app.

Reply:
Presumably, it's just the styling of the scrollbar that doesn't work on Firefox/Edge, but the scrollbar itself still works.

Reply:
Exactly.  The pseudo-element is ignored and the scrollbar has it's default appearance (tested in Firefox and Edge).  Things work fine, the scrollbar just doesn't look as pretty as it does in Michael's spec.

Reply:
We used this approach in Cbio since the consensus was that Chrome was the only browser that we care about and we needed to style the scrollbar.  I still, however, think things should at least function and the UI/appearance should be acceptable in ALL modern browsers (including Edge - even if I'm the only one who uses it!).